src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java

```diff
@@ -92,6 +92,52 @@ public void testSuccessWithIncludeStatusFalse() throws Exception {
     assertIsTartanBank(institution);
   }
 
+  @Test
```

Comment:
Do we need to add these same tests for `InstitutionsGet`?

Reply:
Hard to test PI metadata for `InstitutionsGet` since we cannot force retrieve "Royal Bank of Plaid" easily